rc/extra/kitty.kak

```diff
@@ -22,7 +35,27 @@ If no client is passed then the current one is used} \
         if [ $# -eq 1 ]; then
             printf %s\\n "evaluate-commands -client '$1' focus"
         else
+            # need the no-response switch to get added to focus-tab first
+            # kitty @ focus-tab --no-response -m=id:$kak_client_env_KITTY_WINDOW_ID
             kitty @ focus-window --no-response -m=id:$kak_client_env_KITTY_WINDOW_ID
         fi
+}}
+
+define-command -docstring %{kitty-repl [<arguments>]: create a new window for repl interaction
+All optional parameters are forwarded to the new window} \
+    -params .. \
+    -command-completion \
+    kitty-repl %{ evaluate-commands %sh{
+        if [ -z "${kak_opt_termcmd}" ]; then
+           echo "echo -markup '{Error}termcmd option is not set'"
+           exit
+        fi
+        if [ $# -eq 0 ]; then cmd="${SHELL:-sh}"; else cmd="$@"; fi
```

Comment:
I think that the `$SHELL` variable should contain a fullpath, so the default value here should probably be `/bin/sh`.

As for the `else` statement, I think you want `cmd="$*"` here, which is guaranteed to be portable.